dali/image/tiff_libtiff.cc

```diff
@@ -247,17 +247,19 @@ TiffImage_Libtiff::TiffImage_Libtiff(const uint8_t *encoded_buffer,
   LIBTIFF_CALL(
     TIFFGetField(tif_.get(), TIFFTAG_IMAGEWIDTH, &shape_[1]));
   LIBTIFF_CALL(
-    TIFFGetField(tif_.get(), TIFFTAG_SAMPLESPERPIXEL, &shape_[2]));
+    TIFFGetFieldDefaulted(tif_.get(), TIFFTAG_SAMPLESPERPIXEL, &shape_[2]));
```

Comment:
Can you add a test for it? Maybe add an additional TIFF image with that properites to DALI_extra?

Reply:
Good idea